eoscostarica/guide.eoscostarica.io
# docs(engineering-culture.md): add link whit tips the work for home

### Add links to blog posts from relevant pages

#82 

### Steps to test
1. Run project locally
1. Go to Engineering Culture and see content

#### CheckList
